apps/molgenis-components/src/components/layout/Molgenis.vue

```diff
@@ -66,12 +66,17 @@ export default {
         {
           label: "Schema",
           href: "schema",
-          role: "Manager",
+          role: "Viewer",
```

Comment:
schema viewer ?

Reply:
Yes, schema is useful for somebody with view permission as documentation. It is not only editing of the schema

Reply:
check